src/events/funStuff.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
                 `(?:^|[^a-zA-Z0-9])${keyword}(?:$|[^a-zA-Z0-9])`,
                 "i"
             );
-            if (Math.random() < 0.5 && regex.test(message.content)) {
+            if (Math.random() < Math.random()*Math.random() && regex.test(message.content)) {
```

Comment:
Something I wanted to mention is I ran 
```js
let total = 0;
for (let i = 0; i < 10000; i+=1) {
    let res = Math.random() < Math.random()*Math.random();
    if (res) total++
}
return total/10000
```
a couple of times and it revealed that this is just a fancy way of a 25% chance. Do you think that that is too little?

Reply:
I did some testing and it occurs between every 2-3 messages which seems better idk, honestly I just messed around till the frequency felt right.